lib/improver/metadata/utilities.py

```diff
@@ -31,10 +31,57 @@
 """General IMPROVER metadata utilities"""
 
 import hashlib
+import iris
```

Comment:
One last thing I noticed - I don't know if we're consistent throughout `improver`, but we should move the `import iris` down with the other third party libraries, at least according to PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
I know we tried to be consistent - either Neil or Tom made a recent change to tidy up import orders.  But it is hard to know what the correct order is!  I will make this change, but if we want to maintain this discipline we really need it to be checked for in the automated test suite.

Reply:
Sorry @arh89 , I actually don't know what the correct import order is in this case.  Could you add it in a comment here please?

Reply:
Sorry, only just seen this. If you move iris above dask.array, then that's fine